host/sgx/xstate.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_XSTATE_H
+#define _OE_XSTATE_H
+
+#include <openenclave/internal/sgxtypes.h>
+
+/* Detect the XSave Feature Request Mask (XFRM) to set in the enclave */
+uint64_t _xfrm_detect(void);
```

Comment:
This is a static local function in sgxload.c and does not need to be included in this header (otherwise, it should be prefixed with `oe_`) Please remove from this header.

Reply:
I did not define it as static earlier. Fixed this to be static and also removed from the header. So, conforming to standards now..